src/main/resources/assets/js/app/view/ApplicationItemStatisticsHeader.ts

```diff
@@ -1,9 +1,18 @@
 import {ItemStatisticsHeader} from '@enonic/lib-admin-ui/app/view/ItemStatisticsHeader';
 import {Application} from '@enonic/lib-admin-ui/application/Application';
+import {DivEl} from '@enonic/lib-admin-ui/dom/DivEl';
 
 export class ApplicationItemStatisticsHeader
     extends ItemStatisticsHeader {
 
+    constructor() {
+        super();
+
+        const wrapper: DivEl = new DivEl('titla-path-wrapper');
```

Comment:
`const wrapper: DivEl = new DivEl('titla-path-wrapper');`

Obviously a typo here (`title`, not `titla`). But it doesn't seem like this is used since I didn't find the corresponding CSS style.

Reply:
Updated